src/App.js

```diff
@@ -180,7 +179,9 @@ class Root extends React.Component {
    * Method called when need to reload data when websocket reconnects
    */
   reloadData = (data) => {
-    wallet.reloadData();
+    if (hathorLib.wallet.loaded() && !this.props.loadingAddresses) {
```

Comment:
The same question here: what changed in the lib that we need this if statement?

Reply:
This was already a bug, in my opinion.

The reload is called every time we move to the online state and it was not online before. However, it makes no sense to reload the wallet data before loading it, or while we are loading it.

Reply:
About the first condition (`hathorLib.wallet.loaded()`), I agree. But the second condition should never happen right? What if the connection is down while the wallet is syncing (i.e., loading the transaction address)? Wouldn't it prevent the wallet from starting it over?

Reply:
If the connection is down while the wallet is syncing it will show the error modal with the options:

- Retry request
- Reset wallet
- Change server

The automatically reload data shouldn't be possible while loading the data from the server, only after the wallet is already loaded.

Reply:
I got it. Just to double check: have you tested connection issues to confirm that the wallet can still connect?

Reply:
What do you mean about 'connection issues'? I've tested becoming offline and then online again and the data is reloaded normally.

Reply:
As we discussed in Slack, it is ok. Thanks!